Add the Makefile and Pipfile

jbasden opened the pull request:
This commit adds a basic makefile and pipfile.
It also changes the name of the testing folder from tests to test.

Commit:
Add the Makefile and Pipfile
This commit adds a basic makefile and pipfile.
It also changes the name of the testing folder from tests to test.

ylynn commented on Makefile:
```diff
@@ -0,0 +1,47 @@
+	pipenv sync --dev
+
+compile: dev-deps
+	pipenv run pydocstyle app
```
Change app to metricpublisher.

jlhood commented on Pipfile:
```diff
@@ -0,0 +1,19 @@
+name = "pypi"
+
+[packages]
+"boto3" = "*"
```
Why does boto3 have quotes around it when none of the others do?

jlhood reviewed (approved):
One minor comment. Otherwise, looks good!

Commit:
Add docstrings to empty files to pass linting tests.
When "app" was changed to "metricpublisher", flake8 and pycodestyle gave a bunch of errors,
mostly that there were no docstrings, so I added some temporary docstrings.

Commit:
Change "boto3" to boto3 (without the quotes).
It appears there is no difference, but I took out the quotation marks for consistency.

ylynn reviewed (approved):
Provide some info about test. Note down the command you ran to test, like "make test".
